### DISCUSSION TITLE
1.1.3

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#9635] [General]
*  Fix localhost relay issues introduced in 1.1.0 (solves #9508)
*  Add a "Update Available" toast to WSL
*  Ensure the /init bind mount is readonly
*  Set thread names in WSL init deamon
*  Mask SIGCHILD before forking
*  Correct wslpath's output when '.' is passed (solves #9606)
*  Fix debug shell on arm64
*  Update Microsoft.Windows.ImplementationLibrary to version 1.0.230202.1
*  Set default page reporting order to single-page
*  Cleanup the hostname so it always matches systemd-hostnamed (solves #9391)
*  Update Microsoft.WSL.Kernel to version 5.15.90.1
*  Fix incorrect parsing of /proc/<pid>/stat when the process name has spaces
*  Restore signal handlers in mini_init before executing init

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/1.1.3'>1.1.3</a>.</em>

**Comment by svetoslavenchev** [comment]
Just upgraded, a quick smoke test with Docker Compose shows that issue #9508 is fixed! Thank you! :)

**Reply by Slach** [reply]
why you provide ARM64 artifact, but doens't provide AMD64 artifacts?

**Reply by Biswa96** [reply]
The artifact contains both x64 and arm64 packages in one file.
